src/Controls/src/Core/Handlers/Items/iOS/ItemsViewController.cs

```diff
@@ -231,6 +229,9 @@ public virtual void UpdateFlowDirection()
 
 		public override nint NumberOfSections(UICollectionView collectionView)
 		{
+			if(ItemsSource == null)
```

Comment:
This shouldn't be happening - if the binding to ItemsSource is null, the ItemsSource here should be the EmptySource. And you can't skip the CheckForEmptySource() check; it will break the EmptyView.

Reply:
This happen because Mapper didn't run yet, i can change this . We were using Mapper, and also creating when VIewLoad the ItemsSource.. removing from ViewLoad makes it break here for example